SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -331,8 +331,30 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Cloud", meta = (DisplayName = "Region"))
 		TEnumAsByte<ERegionCode::Type> PrimaryDeploymentRegionCode;
 
+	UPROPERTY(EditAnywhere, config, Category = "Cloud", meta = (DisplayName = "Deployment tags"))
+		FString DeploymentTags;
+
 	const FString SimulatedPlayerLaunchConfigPath;
 
+public:
+	UPROPERTY(EditAnywhere, config, Category = "Assembly", meta = (DisplayName = "Windows Build Type"))
```

Comment:
please add documentation to these like we did for the other UPROPERTY fields

Reply:
this is also not consistent, I assume you mean a comment? and none of the cloud one's have comments, but I'll get this updated.

Reply:
yes I mean comments. And we do add comments for those that are public. if we haven't for some, then that was something that slipped through by mistake